virtcontainers/qemu.go

```diff
@@ -613,6 +613,9 @@ func (q *qemu) virtiofsdArgs(sockPath string) []string {
 		args = append(args, "-f")
 	}
 
+	if q.config.VirtioFSExtraArgs != "" {
+		args = append(args, strings.Fields(q.config.VirtioFSExtraArgs)...)
```

Comment:
Note: This will fail I think for options like `-o arg1="hello world"`.

Reply:
> Note: This will fail I think for options like `-o arg1="hello world"`.

This is an important point since many virtiofsd options use the `-o arg1="hello world"` structure.

Reply:
Hi @stefanha, good point, and it got me here, lol, have no idea how to handle this case clearly. But can you give me an example which virtiofsd option act like `"hello world"`?

I am thinking if we can slice struct for `virtio_fs_extra_args` like:
```
virtio_fs_extra_args = ["-o", "arg1=xxx,arg2", "-o", "hello world"]
```

Reply:
> Hi @stefanha, good point, and it got me here, lol, have no idea how to handle this case clearly. But can you give me an example which virtiofsd option act like `"hello world"`?
> 
> I am thinking if we can slice struct for `virtio_fs_extra_args` like:
> 
> ```
> virtio_fs_extra_args = ["-o", "arg1=xxx,arg2", "-o", "hello world"]
> ```
> 
> Anyway, will it a try later, thanks~

Any option that takes a string that can contain a space.  There are not many, -o source=PATH and -o vhost_user_socket=PATH, are the main ones and they will not be set via virtio_fs_extra_args.  However, I still think this patch should allow for future option strings to contain spaces.